src/main/c/org_broadinstitute_hellbender_utils_bwa_BwaMemIndex.c

```diff
@@ -4,6 +4,19 @@
 #include "jnibwa.h"
 #include "bwa/bwa_commit.h"
 
+#define jStringToConstChar(A,B) \
```

Comment:
https://stackoverflow.com/questions/14041453/why-are-preprocessor-macros-evil-and-what-are-the-alternatives